Update sql-database-managed-instance.md

jboeshart opened the pull request:
Added link in Next Steps section to the VNet Configuration document and link to same document in the security isolation section.

azurecla commented:
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@4c74356b41 : Thanks for your contribution to the Azure documentation! The author, @Rboucher, has been notified to review your proposed change.

rboucher commented:
Thanks for removing these typos @4c74356b41.  I do appreciate the time you took to do that.  I'll take a look at the use of its and it's.  "Its" is the possessive pronoun and should be used in the same context as "the car's tire ".  "It's" is a contraction and short for "it is".

rboucher commented:
#sign-off

PRmerger commented:
Only the author of the article can use the sign off comment to add the ready-to-merge label.

tysonn commented:
@rboucher - Rob, can you please resolve the merge conflict? And, update the metadata to match your actual account. The automation is case sensitive (authors="Rboucher" in article)

4c74356b41 commented:
Hello?

rboucher commented:
Hi. Still here.  Just need to solve that merge conflict on my side.  Will do that soon.

4c74356b41 commented:
Thanks!

rboucher commented:
Hi 4c74356b41.
Because of the size of the article, I'm having a heck of a time with github actually trying to update literally a single character in this pull request. That's blocking me from incorporating your changes.  I think I'm going to have to pull your changes into another pull request of my own and submit them, then report back when it's done.  Is that okay with you?

rboucher commented:
Hi 4c74356b41.
Because of the size of the article, I'm having a heck of a time with github actually trying to update literally a single character in this pull request. That's blocking me from incorporating your changes.  I think I'm going to have to pull your changes into another pull request of my own and submit them, then report back when it's done.  Is that okay with you?

4c74356b41 commented:
I'm okay with whatever way you are doing this ;)

rboucher commented:
Now at https://github.com/Azure/azure-content-pr/pull/28602. That is a private repository so you probably can't view it. I'll follow up when the changes are live.

rboucher commented:
I was on vacation.  These changes were incorporated 12 days ago and are live now.  Thanks!

4c74356b41 commented:
if you say so

jboeshart committed:
Update sql-database-managed-instance.md

Added link in Next Steps section to the VNet Configuration document and link to same document in the security isolation section.
